### PR TITLE
Script for sending DataItems to the Sequencer

### PR DESCRIPTION
**Pull request by janekolszak**
Usage:
```
ts-node send-dataitem.ts --tag 1=asdf --tag fdasfa=234 --arweave-wallet ./arweave.json --data data --address localhost
```
or for MetaMask in the browser:
```
cd tools/sequencer/web
npx webpack serve
```

### CODE REVIEW

**Comment by ppedziwiatr** [comment]
hmm, build fails with
image

File: tools/sequencer/send-dataitem.ts [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        const [name, value] = t.split('=')
+        tags.push({ name, value })
+    }
+
+    return createData(fs.readFileSync(args.data), signer, { tags })
+}
+
+function send({ body, ...options }) {
+    return new Promise((resolve, reject) => {
+        const req = http.request({
```
[the remark this answers is not in the document]
Comment (szynwelski) [reply tools/sequencer/send-dataitem.ts]: changed to fetch

File: tools/sequencer/send-dataitem.ts [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        req.end();
+    })
+}
+async function main() {
+    const args = commandLineArgs(optionDefinitions)
+
+    const dataItem = createDataItem(args)
+
+    send({
+        hostname: args.address,
```
[the remark this answers is not in the document]
Comment (szynwelski) [reply tools/sequencer/send-dataitem.ts]: added `Content-Type` header

**Comment by szynwelski** [comment]
Script moved to #442.

**Comment by szynwelski** [comment]
I moved here the script to send data item directly to sequencer (#423) without using SDK

**Pull request closed without merging**